src/Silverback.Integration.Kafka/Messaging/Broker/KafkaBroker.cs

```diff
@@ -13,6 +13,8 @@ namespace Silverback.Messaging.Broker
     /// </summary>
     public class KafkaBroker : Broker
     {
+        internal const string MessageKeyHeaderKey = "x-kafka-message-key";
```

Comment:
Would it make sense to put this const somewhere else and make it public? According to the issue the intention is to use it from outside, isn't it?

Reply:
Oops, yes, that should definitely be public now. Thanks!

Reply:
Moved all constants into a separate class (public of course).